### DISCUSSION TITLE
Validation of some International channels

### DISCUSSION THREAD

**Discussion by fershady** [iptv-org/community#936] [Show and tell]
I believe that some channels take advantage of international categorisation.

For instance, looking at the various international channels found in the "Nicaragua" playlist and how they are currently shown in the database, I suggest the following:

- **Angel TV Spanish**, **Angel TV America**: This channel is in foreign language. 

- **Filmstream**, **Golden TV**: This channel is in english language.

- **Galicia TV América**: It is in the Galician language that is only spoken in the  northwestern Spain.

- **MBC America**: This channel is in Korean language.

All broadcasts on the Internet are not international channels. 

- **Bandamax**, **Vallenato Internacional**: These music style belongs to a specific ethnic group, Mexican and Colombian respectively. 

- **CCP Radio** (not 24/7): It  is not even a channel. It is an online radio from Chile.

- **Las Estrellas Latin America**: Mexican content.

- **TV Daja**: Dominican Republic content.

- **TV Perú Internacional**: Peru content.

- **Radio San Borja TV**: presenting their country does not seem to me to be an international channel but an obvious sign of nationalism.

- **Nuevo tiempo**, **Restaurando Vidas Internacional**: Religious channels take advantage of this international category. Even if the content can be shared with an audience of the same language, international channels should nevertheless have an accent that is accessible to all audiences, a so-called neutral accent. The accent and regionalism (lexicon of a region) can be a barrier for people of the same language.

And finally, by definition, any international channel must broadcast for 24/7.

**Comment by freearhey** [comment]
International channels have been moved to a separate playlist in https://github.com/iptv-org/iptv/pull/26208

If you find any other errors in the channel description, please open a new issue here: https://github.com/iptv-org/database/issues
